documentation/dev/introduction.rst

```diff
@@ -96,6 +120,16 @@ Or:
 
 And access the server at http://localhost:5000
 
+If you added a toy account, you could log in with `toy-user@flexmeasures.io`, password `toy-password`.
+
+Otherwise, you need to add some other user first. Here is how we add an admin:
+
+.. code-block:: bash
+    flexmeasures add account --name MyCompany   
```

Comment:
This code block is not showing to me locally. Trying, I manage to make it appear putting an empty line above. Also, I think we should put the dollar sign in front.